include/llvm/IR/Type.h

```diff
@@ -71,7 +71,8 @@ class Type {
     StructTyID,      ///< 12: Structures
     ArrayTyID,       ///< 13: Arrays
     PointerTyID,     ///< 14: Pointers
-    VectorTyID       ///< 15: SIMD 'packed' format, or other vector type
+    VectorTyID,      ///< 15: SIMD 'packed' format, or other vector type
+    UnionTyID        ///< 16: Unions
```

Comment:
This seems unnecessary. We shouldn't need a new union type in the LLVM IR layer, and I don't think this is even used.